applications/CoSimulationApplication/tests/fsi_wall/cosim_wall_weak_coupling_fsi_parameters.json

```diff
@@ -4,7 +4,8 @@
         "start_time" : 0.0,
         "end_time" : 3.0,
         "echo_level" : 0,
-        "print_colors" : true
+        "print_colors" : true,
```

Comment:
As a side note. At some point @loumalouomega  was mentioning about some color printing in the core. may be we can switch to that when it is available. (for the kratos version)

Reply:
he was referring to [this](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/color_utilities.h), but it is not really integrated yet / exposed to python, hence I am not using it yet
I agree about using the one from the core once available :+1: